commercetools-models/src/main/java/io/sphere/sdk/carts/CartLike.java

```diff
@@ -204,6 +204,14 @@ default CurrencyUnit getCurrency() {
     @Nullable
     Locale getLocale();
 
+
+    /**
+     *  The shippingRateInput is used as an input to select a shipping rate price tier
+     * @return shippingRateInput
+     */
+    @Nullable
+    ShippingRateInput getShippingRateInput();
```

Comment:
This common interface for `Cart`and `Order`shouldn't have this property, because `Order` doesn't have this property. Please remove this method.

Reply:
Actually this property should be added for order too 
release note mentions 
* "[API] Added the `shippingRateInput` field on the [Order](/http-api-projects-order.html#order)".